### PR TITLE
Add support for custom option rendering to Dropdown

### PR DESCRIPTION
**Pull request by sferg-msft**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ X] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ X] Documentation update

#### Description of changes

Dropdown has a custom selected item renderer, but it does not have a custom option renderer.  I would like to add a custom option renderer so that I can render options as something more than just text.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -183,6 +183,11 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
   }
 
   @autobind
+  private _onRenderOption(item: IDropdownOption): JSX.Element {
+    return <span>{ item.text }</span>;
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: can't you just return `{item.text}` here?

Comment (sferg-msft) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: No, that does not compile.  It has to be a JSX.Element.

**Comment by sferg-msft** [comment]
@cschleiden Is there a proper way to generate the change file?  I ran npm run change and it said no change file is needed.

**Comment by cschleiden** [comment]
@sferg-msft I'm not sure :) you could also try running `rush` directly?

**Pull request merged**